lib/maths/CBoostedTreeLeafNodeStatistics.cc

```diff
@@ -134,7 +134,8 @@ CBoostedTreeLeafNodeStatistics::split(std::size_t leftChildId,
                                       const TImmutableRadixSetVec& candidateSplits,
                                       const TSizeVec& featureBag,
                                       const CBoostedTreeNode& split,
-                                      bool leftChildHasFewerRows) {
+                                      bool leftChildHasFewerRows,
+                                      TNodeVec& tree) {
```

Comment:
All the changes in this file should be reverted now.

Reply:
Reverted in c30908e0